Kitodo-SchemaConverter/src/main/java/org/kitodo/schemaconverter/SchemaConverter.java

```diff
@@ -16,6 +16,8 @@
 import org.kitodo.api.schemaconverter.SchemaConverterInterface;
 import org.kitodo.api.schemaconverter.SchemaFormat;
 
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
Why did you add a internal, proprietary and possible deprecated class if we have your own class `org.kitodo.exceptions.NotImplementedException`?